packages/@okta/vuepress-site/docs/reference/api/oie-policy-types/index.md

```diff
@@ -2056,3 +2056,143 @@ A Mapping object links together an Application and a Policy.
   }
 }
 ```
+
+## Updates to Password Policy
+
+This updated Password Policy API shows only the delta from the original [Password Policy API](/docs/reference/api/policy/#password-policy)
+
+Earlier, we supported setting recovery factors on the [policy](/docs/reference/api/policy/#policy-settings-example-2). While we will still continue to support it for backwards compatibility of older clients, the new password policy model does not contain recovery factors. This is now moved to the password policy rule model.
+
+### Password Policy Settings Example
+
+```json
+"settings": {
+     "password": {
+       "complexity": {
+         "minLength": 8,
+         "minLowerCase": 1,
+         "minUpperCase": 1,
+         "minNumber": 1,
+         "minSymbol": 0,
+         "excludeUsername": true
+       },
+       "age": {
+         "maxAgeDays": 0,
+         "expireWarnDays": 0,
+         "minAgeMinutes": 0,
+         "historyCount": 0
+       },
+       "lockout": {
+         "maxAttempts": 10,
+         "autoUnlockMinutes": 0,
+         "showLockoutFailures": false
+       }
+     },
+     "delegation": {
+       "options": {
+         "skipUnlock": false
+       }
+     }
+}
+```
+
+Recovery Factors have been moved from the Policy Settings object to the Password Policy Rule object, specifically inside the `selfServicePasswordReset` Action.
+
+Below are 3 examples of how recovery factors are configured on the rule based on admin requirements.
+
+### Password Policy Rule Action Examples
+
+In this example, the requirement is that end users verify with just one authenticator before they can recover their password. Email, SMS, Voice or Okta Verify Push can be used by end users to initiate recovery. We know that only one authenticator is required because there are no step up authenticators specified as can be seen by `stepUp` object containing the `required` attribute set as `false`.
```

Comment:
```suggestion
In this example, the requirement is that end users verify with just one Authenticator before they can recover their password. Email, SMS, Voice or Okta Verify Push can be used by end users to initiate recovery. We know that only one Authenticator is required because there are no step up Authenticators specified as can be seen by the `stepUp` object having the `required` attribute set as `false`.
```